presto-main/src/test/java/com/facebook/presto/operator/scalar/TestColorFunctions.java

```diff
@@ -130,6 +130,7 @@ public void testRenderDouble()
             throws Exception
     {
         assertEquals(render(1234.5678, color(toSlice("red"))), toSlice("\u001b[38;5;1m1234.5678\u001b[0m"));
+        assertEquals(render(1234.5678f, color(toSlice("red"))), toSlice("\u001b[38;5;1m1234.5677490234375\u001b[0m"));
```

Comment:
That's probably not correct output from float function.
Looks like you gave a float input to a double function and read an output from it to put here as an expected output.